book/src/SDK/platform/mongodb/overview.md

```diff
@@ -0,0 +1,44 @@
+## MongoDB
+
+The MongoDB module is an opinionated micro-framework for interacting with the MongoDB backend that 
+Harbor currently supports. This module was developed based on the following needs and desires:
+
+- At the time of initial development there did not seem to be a widely-used or well-maintained 
+  Rust-based Object Document Mapper (ODM) available from the ecosystem. Database mapping code is 
+  inherently full of boilerplate, which we equate to engineering toil, and can also be a 
+  significant source of bugs. While the Rust type system helps with the latter issue to a great 
+  degree, we still wanted to leverage generics to generate as much boilerplate code as possible.
+- The official Mongo Rust Driver (see below) is wonderful, but also deep and finely-grained. 
+  This is appropriate since it is a low-level driver designed to allow total control of 
+  all aspects of MongoDB interaction. We wanted a way to abstract away that complexity and allow 
+  developers to focus on domain logic rather than repeatedly handle infrastructure concerns.
+- We wanted database interactions to be consistent and easily refactorable in a handful of places 
+  rather than scattered throughout the codebase.
+- The Mongo Rust Driver is designed in a very specific way in terms of [connection management](https://mongodb.github.io/mongo-rust-driver/manual/performance.html).
```

Comment:
Here we have mention of the Mongo Rust Driver where there is an entire section dedicated to it below.  Can we move all of the explantation to there to keep Mongo Rust Driver content easy to find?